db/large_data_handler.cc

```diff
@@ -58,6 +58,7 @@ void large_data_handler::start() {
 future<> large_data_handler::stop() {
     if (running()) {
         _running = false;
+        large_data_logger.info("Waiting for {} background handlers", max_concurrency - _sem.available_units());
```

Comment:
nit, might want to s/exist/exit/ in the commit message.

Reply:
i am curious how can you tell if the background handler exit? shall we also print something after `_sem.wait()` resumes / returns ?

Reply:
Well, in _this _particular case when sem.wait exists, the whole .stop() finishes and database stopping code starting printing messages from other parts

Reply:
ahh, i see. thanks! no wonder.